### PR TITLE
fix: do not rewrite config for lnds

### PR DESCRIPTION
**Pull request by erkarl**
This commit adds a check to avoid rewriting lnd's configuration file if it already exists.

### CODE REVIEW

File: images/lnd/entrypoint.sh [hunk 1]
```diff
@@ -9,12 +9,14 @@ cd "$SCRIPT_PATH" || exit 1
 LND_DIR="/root/.lnd"
 mkdir -p $LND_DIR
 
-if [ "$CHAIN" = "bitcoin" ]; then
-  echo "[DEBUG] Using configuration for bitcoin"
-  cp /root/lnd-btc.conf $LND_DIR/lnd.conf
-else
-  echo "[DEBUG] Using configuration for litecoin"
-  cp /root/lnd-ltc.conf $LND_DIR/lnd.conf
+if [[ $LND_REWRITE_CONFIG || ! -e $LND_DIR/lnd.conf ]]; then
```
Comment (sangaman) [inline images/lnd/entrypoint.sh]: Where does `$LND_REWRITE_CONFIG` get set? I'm not seeing it anywhere else in the codebase.

Comment (erkarl) [reply images/lnd/entrypoint.sh]: It's mainly for testing/debugging purposes. You can set it in `docker-compose.yml` under lnd->environment.

Comment (erkarl) [reply images/lnd/entrypoint.sh]: I removed it since in hindsight it doesn't provide much value.

Comment (sangaman) [reply images/lnd/entrypoint.sh]: Gotcha, I think it could be fine to have as well but not too important either way.

**Pull request merged**